erpnext/accounts/report/asset_depreciations_and_balances/asset_depreciations_and_balances.py

```diff
@@ -71,46 +71,47 @@ def get_asset_categories_for_grouped_by_category(filters):
 		condition += " and asset_category = %(asset_category)s"
 	return frappe.db.sql(
```

Comment:
Linters failing, use `# nosemgrep` if the `frappe.db.sql` cannot be replaced with frappe ORM  or `frappe.qb`.